datadog_lambda/wrapper.py

```diff
@@ -138,15 +148,18 @@ def _before(self, event, context):
         except Exception:
             traceback.print_exc()
 
-    def _after(self, event, context):
+    def _after(self, event, context, response):
         try:
             if not self.flush_to_log or should_use_extension:
                 lambda_stats.flush(float("inf"))
             if should_use_extension:
                 flush_extension()
 
             if self.span:
+                set_http_status_code_tag(self.span, response)
                 self.span.finish()
+                # Close the DD subsegment here and send it
+            close_dd_subsegment()
```

Comment:
@tianchu I updated this to end the DD subsegment in the `_after` block so that we can pass the `status_code` tag to the existing subsegment. Is that the right way to go about it?